RPIPs/RPIP-46.md

```diff
@@ -29,91 +29,121 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 
 ## Specification
 ### UARS
-1. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `node_operator_commission_share_council_adder`, `voter_share`, `surplus_share`
+1. There SHALL be the following defined shares with settings: `node_operator_commission_share`, `node_operator_commission_share_council_adder`, `voter_share`
    1. `node_operator_commission_share + node_operator_commission_share_council_adder`: each NO receives this percentage of the commission from the borrowed ETH on validators they run. Unlike the remainder of the shares, this is _not_ a protocol revenue (ie, it is not socialized).
-   2. `voter_share`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
-   3. `surplus_share -  node_operator_commission_share_council_adder`: this share of revenue is used to distribute revenue beyond that used for protocol operation (such as the shares above)
-2. `reth_commission` SHALL be defined as the sum of `node_operator_commission_share`, `voter_share`, and `surplus_share`
+   2. `voter_share -  node_operator_commission_share_council_adder`: each NO receives a share of revenue based on the vote-eligible RPL staked to their megapool. The overall voter share of revenue is based on the setting, and each NO receives a proportion of that based on `vote_eligible_RPL_in_their_megapool/total_vote_eligible_RPL_in_megapools`.
+2. `reth_commission` SHALL be defined as the sum of `node_operator_commission_share`, and `voter_share`
 3. `reth_share` SHALL be defined as `100% - reth_commission`
 4. Distributions of revenue from borrowed ETH MUST respect the defined shares
    1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
    2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
-5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+5. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `voter_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
    1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
 6. The `node_operator_commission_share_council_adder` setting SHALL only allow values where:
    1. 0% ≤ `node_operator_commission_share_council_adder` ≤ `max_node_operator_commission_share_council_adder` 
```

Comment:
```suggestion
   1. `node_operator_commission_share_council_adder` ≤ `max_node_operator_commission_share_council_adder` 
   2. `node_operator_commission_share_council_adder` > 0%
```
Just more readable + less hard to conceptualize this way. I'm aware there is a small logical difference in that `max_node_operator_commission_share_council_adder` can now be less than zero, but this is irrelevant as the used value could then not be changed to an invalid value. 

If you want to keep it could add a third line, or just stick with the three-term expression.

Reply:
I find the original more readable. Not planning to change this, unless we hear back from other folks

Reply:
You're both smart AF, am not surprised you're fine. I think others are more likely to struggle. I changed  0 <= X to X > 0 which should be equivalent?